FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -313,21 +326,26 @@ uint8_t ucMinAgeFound = 0U;
 				/* Both the MAC address as well as the IP address were found in
 				different locations: clear the entry which matches the
 				IP-address */
-				memset( &xARPCache[ xIpEntry ], '\0', sizeof( xARPCache[ xIpEntry ] ) );
+				/* Silence MISRA warning about unused return value */
```

Comment:
Delete this comment.  You're casting to `void` here because you're not interested in the return value (i.e. it's the right thing to do), not because you want to get rid of a MISRA warning.

Reply:
Please don't remove them - they are necessary to demonstrate you are intentionally uninterested in the return value - and is the only way of demonstrating that to an automated tool that won't read the comments.

Reply:
+1 on this comment from @RichardBarry.  Every use of a function should be according to its prototype; if that function returns something and you are not interested in it, you should cast to `void`.

Reply:
Sure thing. I added back the (void) to all instances of memmove/memcpy etc.